spec/factories/grda_warehouse/hud/clients.rb

```diff
@@ -1,5 +1,10 @@
 FactoryBot.define do
+  trait :standard_age do
+    DOB { (Date.current - 24.years).to_fs(:db) }
+  end
```

Comment:
could add other traits to dry up the factories but doing the most minimal thing to get tests passing